src/main/webapp/data-visualization.js

```diff
@@ -200,22 +200,18 @@ function getMapsData(censusDataArray) {
   const populationsList = [];
   // Get rid of the header
   const censusArray = censusDataArray.slice(1);
-  censusArray.forEach( (county) => {
+  censusArray.forEach((county) => {
     // The current county strings are in a layout like this:
     // "Contra Costa County, California"
     // and we need to get them like this "Contra Costa"
     const countyAndStateArray = county[0].split(',');
     // ^^ ["Contra Costa County", "California"]
-    const countyArray = countyAndStateArray[0].split(' ');
-    // ^^ ["Contra", "Costa", "County"]
-    let countyString = '';
-    let i;
-    // Get all strings except for the last one
-    for (i = 0; i < countyArray.length - 1; i++) {
-      countyString += countyArray[i];
-      if (i !== countyArray.length - 2) {
-        countyString += ' ';
-      }
+    // Alaska has suffixes other than county
+    const countyRegex = new RegExp('( County| City and Borough|' +
```

Comment:
Maybe add a comment that the spaces are to get rid of the preceding space?  Just so no one sees it later and thinks "oh I'll clean that up and delete the extra spaces"